src/components/Safeplace/Safeplace.css

```diff
@@ -0,0 +1,24 @@
+.Safeplace-list {
```

Comment:
Because of the need to create multiple different lists, I will be working to create a List template.